Makefile

```diff
@@ -1,5 +1,6 @@
 .SECONDEXPANSION:
 
+BEST:=$(shell if which ocamlopt > /dev/null; then echo native; else echo byte; fi)
```

Comment:
In shell, it seems that the error message in case of a missing command is printed on stderr. So the following is better (tested with bash and dash):

    BEST:=$(shell if ocamlopt > /dev/null 2>&1; then echo native; else echo byte; fi)